<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_gamultifit</class>
 <widget class="QDialog" name="dlg_gamultifit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>501</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simultaneous Multi-Domain Fitting</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_pRunGa">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget2">
     <property name="minimumSize">
      <size>
       <width>330</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Basic Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="m_pResolutionLabel">
            <property name="text">
             <string>Resolution of Map:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="m_pResolution">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>15</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Optimization</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="m_pPopSizeLabel_3">
            <property name="text">
             <string>Population Size:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_pPopSize">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>50000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_pGenLabel_2">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Generations:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="m_pMaxGen">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>25</number>
            </property>
            <property name="value">
             <number>3500</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="m_pShowFitnessCurve">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Show Fitness Curve</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Multi-Thread</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="m_pThreadsLabel_2">
            <property name="text">
             <string>Threads:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_pMaxThread">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_pSyncGenLabel_2">
            <property name="text">
             <string>Generations/Parallel Run:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="m_pSyncGen">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Score function</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="_7">
            <item>
             <spacer name="spacer2_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="m_pUseComputedCV">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Use Precomputed Vectors</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="m_pCVScore">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Feature Based Scoring</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Advanced Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Tabu Search</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="m_pWindowSizeLabel">
            <property name="text">
             <string>Window Size:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="m_pTabuWindowSize">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string>30</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_pThreasholdLabel">
            <property name="text">
             <string>Threshold:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="m_pTabuThreshold">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string>0.5</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_pRegionSize">
            <property name="text">
             <string>Region Size:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="m_pTabuRegionSize">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string>3.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Score Penalties</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="m_pGeneDistanceLabel">
            <property name="text">
             <string>Distance:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="m_pDistanceThreshold">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string>10.0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_pPenaltyLabel">
            <property name="text">
             <string>Penalty:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="m_pDistancePenalty">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string>0.9</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Sampling Around Centers</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QCheckBox" name="m_pSample">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Sample Population</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_pPopAroundCenters">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>150</height>
             </size>
            </property>
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>50000</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="title">
       <string>Output</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Output and Statistics</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0" rowspan="2" colspan="3">
           <layout class="QHBoxLayout" name="_10">
            <item>
             <widget class="QLabel" name="m_pLocation">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Directory</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_pOutputPath">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_pSelectOutputDir">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="m_pOutput">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Write Solutions To Disk</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="m_pPopSizeLabel_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Write Output Interval:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="m_pWriteModelInterval">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>261</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>tabWidget2</tabstop>
  <tabstop>m_pRunGa</tabstop>
  <tabstop>pushButton1</tabstop>
 </tabstops>
 <includes>
  <include location="local">sculptor_document.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton1</sender>
   <signal>clicked()</signal>
   <receiver>dlg_gamultifit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pRunGa</sender>
   <signal>clicked()</signal>
   <receiver>dlg_gamultifit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
